configure.ac

```diff
@@ -231,6 +231,8 @@ RKT_ITERATE_FLAVORS([${RKT_STAGE1_FLAVORS}],[flavor],
                                       RKT_REQ_PROG([BC],[bc],[bc])],
                              [host],
                                      [],
+                             [fly],
+                                     [],
```

Comment:
I would put `AC_MSG_WARN([* fly is an experimental stage1 implementation, some features are missing])` between the square brackets.